xc/common/utils/prjxray_form_channels.py

```diff
@@ -578,6 +582,15 @@ def import_phy_grid(db, grid, conn, get_switch, get_switch_timing):
                 )
             )
 
+            site_instance_pkey = write_cur.lastrowid
+
+            cur.execute(
```

Comment:
This should be "write_cur" not "cur", and then line 490 is not required.

Reply:
Sure, forgot about that. Done